pkg/api/controller.go

```diff
@@ -3511,9 +3511,13 @@ func (c *Controller) GetSetupState(w http.ResponseWriter, r *http.Request) {
 		writeError(w, r, http.StatusInternalServerError, err)
 		return
 	}
+
+	if savedState == auth.SetupStateNotInitialized {
+		c.Collector.CollectEvent(stats.Event{Class: "global", Name: "preinit", Client: httputil.GetRequestLakeFSClient(r)})
```

Comment:
This is a new metric you understand how many users are still in preinit state?
The call for GetSetupState is called many names - each login if I remember to understand if we need to redirect the user to the setup flow.
The problem will be that for external API we always in not initialized mode and we will get this event on all checks.